src/components/calcite-accordion/calcite-accordion.tsx

```diff
@@ -21,26 +21,24 @@ export class CalciteAccordion {
   //
   //--------------------------------------------------------------------------
 
-  /** specify the theme of accordion, defaults to light */
-  @Prop({ reflect: true }) theme: "light" | "dark";
-
-  /** specify the scale of accordion, defaults to m */
-  @Prop({ mutable: true, reflect: true }) scale: "s" | "m" | "l" = "m";
-
   /** specify the appearance - default (containing border), or minimal (no containing border), defaults to default */
-  @Prop({ mutable: true, reflect: true }) appearance: "default" | "minimal" | "transparent" =
-    "default";
+  @Prop({ reflect: true }) appearance: "default" | "minimal" | "transparent" = "default";
 
   /** specify the placement of the icon in the header, defaults to end */
-  @Prop({ mutable: true, reflect: true }) iconPosition: "start" | "end" = "end";
+  @Prop({ reflect: true }) iconPosition: "start" | "end" = "end";
```

Comment:
Noticing the removal of `mutable`. Are we sure we can remove that? Applies to other removals.

Were these just set to mutable when they shouldn't be? I don't see them being set from inside the shadowDom with the constructor updates.

Reply:
Yep, these can be removed since they are no longer changed internally. The linter warns about this actually.